### PR TITLE
Nav logic

### PR DESCRIPTION
**Pull request by lawhump**
Nav logic and small jquery like selector for cleaning up code

### CODE REVIEW

**Comment by bakerTX** [comment]
did we replace "results" with "anything" ?

looks good! 👍

File: src/js/index.js [hunk 1]
```diff
@@ -1,9 +1,19 @@
+var $ = function (query) {
+  var res = document.querySelectorAll(query);
+
+  if (res.length === 1) {
+    return res[0];
+  }
+
+  return res;
+};
```
Comment (electrostaticfleece) [inline src/js/index.js:15]: If we have n number of elements on a page where n >= 1, does this return either a NodeList in the event that n > 1 or a Node if n === 1? If the NodeList returned from querySelectorAll could be either 1 or greater than 1 does this return two different interfaces, a Node or a NodeList respectively? If I'm not mistaken, I think doing so would force us to check the length property on the returned value, and write a conditional that handles both situations if we do not definitively know that there is only one node in the DOM or many nodes for a given query

Comment (lawhump) [reply src/js/index.js:15]: Yeah, it's a Node if n === 1 and a NodeList if n > 1. I use it as shorthand and find it extremely helpful, as you can often intuit if your selector matches one or multiple Nodes.

Comment (lawhump) [reply src/js/index.js:15]: It's near identical to the jQuery functionality.

**Comment by electrostaticfleece** [comment]
Could you comment out the code to let us know two different interfaces are returned, to always check for the existence of the length property if we plan on using this on dynamically generated elements, and to write conditions for both outcomes?

**Comment by lawhump** [comment]
I don't get what you're asking, but do what you need to do. I would like to know how to make that function more intuitive though. It seems pretty straightforward to me, but I wrote it obviously.

**Comment by electrostaticfleece** [comment]
The code is intuitive. My concern is that misuse of the function can lead to bugs because properties like forEach and length don't exist on the Node interface. In the event one of us wants to use this function to select something like all of the charity links, then we should know to write code to cover the case where only one charity was returned based on the quiz results.

**Comment by lawhump** [comment]
Couldn't you cover the cases where you _need_ an array by using `document.querySelectorAll()` though? As a whole, I think the case of saving man-hours and writing more expressive code is a compelling one.

**Review by lawhump — COMMENTED** [review]
ifuxwiddit. Although, you could still use the function in dynamic settings with just a _little_ more tedium. :P

**Review by electrostaticfleece — APPROVED** [review]

**Pull request merged**